imaginary.go

```diff
@@ -53,10 +54,11 @@ const usage = `imaginary %s
 
 Usage:
   imaginary -p 80
-  imaginary -cors
+  imaginary -cors -no-form
```

Comment:
@h2non is this `-no-form` was original idea but then you switch to a better approach with `disable-endpoints` idea and forgot to remove it from docs?

Reply:
That was a typo. It's fixed.